threescale_api/resources.py

```diff
@@ -597,12 +597,25 @@ def url(self) -> str:
         return self.threescale_client.admin_api_url + '/registry/policies'
 
 
-class ProviderAccounts(DefaultStateClient):
+class ProviderAccounts(DefaultClient):
```

Comment:
If I understand this correctly, this is a rename (ProviderAccounts -> ProviderAccountUsers), does this affect the test suite in any way?

Reply:
Currently, it is used only by the permissions UI test, which is not merged.